json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9-]+$"
+    },
+    "frameworkStartDate": {
```

Comment:
I'd prefer to keep this framework agreement data dumb - i.e. it's just a collection of the exact strings we want to patch into the agreement document template.  

It feels like overkill to start saving actual timestamps and having the script to generate agreements know where to find the various timestamps and how they should be formatted for different placeholders in the template.  And what if the format changes from one framework agreement to the next?  We still need to be able to generate the right thing for all frameworks.  Just storing the strings in a convenient blob feels like the best thing.